extensions/spring-cloud.md

```diff
@@ -1,51 +1,89 @@
 # Spring Cloud
 
-Spring Cloud is an alternative approach to distributing command bus \(commands\), besides Axon Server which is the default.
+Spring Cloud is an alternative approach to distributing the command bus \(commands\), besides Axon Server as the default.
 
-The Spring Cloud connector setup uses the service registration and discovery mechanism described by [Spring Cloud](https://spring.io/projects/spring-cloud) for distributing the command bus. You are thus left free to choose which Spring Cloud implementation to use to distribute your commands. An example implementations is the Eureka Discovery/Eureka Server combination.
+The Spring Cloud Extension uses the service registration and discovery mechanism described by [Spring Cloud](https://spring.io/projects/spring-cloud) for distributing the command bus. 
+You thus have the choice of Spring Cloud implementation to use when discovering the routes to distribute your commands. 
+An example implementations is the Netflix' [Eureka Discovery/Eureka Server](https://cloud.spring.io/spring-cloud-netflix/multi/multi__service_discovery_eureka_clients.html) combination or HashiCorp's [Consul](https://www.consul.io/use-cases/service-discovery-and-health-checking).
 
 To use the Spring Cloud components from Axon, make sure the `axon-springcloud` module is available on the classpath.
+Easiest would be to include the Spring Cloud starter (`axon-springcloud-spring-boot-starter`) from this extension to your project.
 
-> **Note**
->
-> The `SpringCloudCommandRouter` uses the Spring Cloud specific `ServiceInstance.Metadata` field to inform all the nodes in the system of its message routing information. It is thus of importance that the Spring Cloud implementation selected supports the usage of the `ServiceInstance.Metadata` field. If the desired Spring Cloud implementation does not support the modification of the `ServiceInstance.Metadata` \(e.g. Consul\), the `SpringCloudHttpBackupCommandRouter` is a viable solution. See the end of this chapter for configuration specifics on the `SpringCloudHttpBackupCommandRouter`.
+Giving a description of every Spring Cloud implementation would push this reference guide to far. 
+For information on any of the Spring Cloud implementation options out there, we refer to their respective documentations.
```

Comment:
A bit confused by this sentence... Are you actually going to talk about every single SC implementation there is in this PR?